riscv-sbi.adoc

```diff
@@ -330,9 +330,18 @@ value for this CSR.
 
 == Legacy Extensions (EIDs #0x00 - #0x0F)
 
-The legacy SBI extensions ignores the SBI function ID field, instead being
-encoded as multiple SBI extension IDs. Each of these extension IDs must be
-probed for directly.
+The legacy SBI extensions follow a slightly different binary encoding as
+compared to SBI v0.2 (or higher) where:
+
+* The SBI function ID field in `a6` register is ignored because these are
+  encoded as multiple SBI extension IDs.
+* Nothing is returned in `a1` register.
+* All registers except `a0` must be preserved across an SBI call by the
+  callee.
+* The value returned in `a0` register is SBI legacy extension specific.
+
+The page and access faults taken by the SBI implementation while accessing
+memory on behalf of the supervisor are redirected back to the supervisor.
```

Comment:
Crucially _with sepc referring to the ecall instruction_, as opposed to the actual load/store in M-mode

Reply:
(which makes the traps look extremely weird, and means the calls must be resumable)

Reply:
Okay, I will add details about sepc referring to the ecall instruction